csrc/options.cpp

```diff
@@ -164,27 +164,57 @@ std::unordered_map<EnableOption, std::vector<std::string>> Options<
   return parseEnvOptions("ENABLE", available_options);
 }
 
+const std::unordered_map<std::string, std::pair<DisableOption, std::string>>&
+getDisableOptionDescriptions() {
```

Comment:
Included the comments as description strings here so that we can fill out an informative docstring.